docs/tutorial.md

```diff
@@ -54,19 +53,19 @@ spec:
     name: echo-hello-world
 ```
 
-To apply the yaml files, use the following command:
+Apply your YAML files to the build as follows:
```

Comment:
I prefer the original wording. Was there a reason to you that this was not clear?

Reply:
A documentation best practice is to always begin a step with an action, hence "Apply." You also want to be consistent with the rest of the document in that regard.

Reply:
Maybe then `to the build` is throwing this off? There isn't really a build concept for Tekton. Perhaps it could be `Apply your YAML file using the following command:`

Reply:
Yeah, I agree. Let me remove "to the build."